src/documents/activities/WebOfStrings.html.md.handlebars

```diff
@@ -7,33 +7,44 @@ category: Activities
+Work together as a team to transfer the object from one container to the other one without dropping it, by collaboratively lifting and moving the web of strings.
 
-Tip: If there are members in the group that require audio description, make sure to describe the direction of movement, the state of the object, the distance to the other container, and anything else that might help them to participate fully.  
+**Tip**: If there are members in the group that require audio description, make sure to describe the direction of movement, the state of the object, the distance to the other container, and anything else that might help them to participate fully.
 
 ## Why
 This activity helps to make the experience of working collaboratively toward a common goal more tangible. The task requires that no one person overpower the others. Participants learn how to move in unison to keep the object from falling or rolling off the web, focusing on moving to the destination while working closely with each other.
 
-
 <figure>
 [![A diagram showing 8 evenly-spaced straight lines that all intersect at their mid-point, creating a kind of giant asterisk or starburst. The lines are labeled with numbers 1 through 8. Underlying the intersection point there is a circle labeled "first container" and a short distance away there is a second circle labeled "second container". Lying on top of the intersection point is a smaller circle labeled "object". Text reads "8 strings set up for 16 participants".  ](/images/web_of_strings_diagram.png)](/images/web_of_strings_diagram.png)
 <figcaption>
 *Figure 1:* Preparatory layout for the web of strings activity. Each participant controls one end of a string, either by holding it in their hand, attaching it to their wheelchair, or in whatever way works best for them. The object is collaboratively transferred from the first container to the second using only the web of strings.
 </figcaption>
 </figure>
+
+<figure>
+[![An image showing a several blue strings laid out on the floor, all intersecting at their midpoint and forming a web in the shape of a large asterisk. The central intersection point of the strings lies on top of a white bowl, and an orange ball lies on top of the intersection point of the strings. Nearby sits another white bowl of the same size.](/images/web_of_strings_layout.jpg)](/images/web_of_strings_layout.jpg)
+<figcaption>
+*Figure 2:* Preparatory layout for the web of strings activity.</figcaption>
+</figure>
+
+<figure>
+[![An image showing a closeup view of the intersection point of the web of strings raised above the floor, on which balances an orange ball. On the floor below a white bowl can be seen, as well as several participants standing in the background holding the ends of the strings.](/images/web_of_strings_closeup.jpg)](/images/web_of_strings_closeup.jpg)
```

Comment:
I don't know - I think the idea is that it is meant to be done this way, but that if the group decides otherwise that's ok too.

Reply:
@sepidehshahi what do you think?

Reply:
I think having a combination of the illustration and the image would be very helpful. The illustration provides a visual instruction, and the image shows the activity in progress. We have also tried to emphasize in the text that these activities are flexible, so groups can change the rules and redefine the structure of the game, the same thing that we did.

Reply:
@michelled @sepidehshahi there's no text in this activity at the moment that reflects the idea of changing the rules...perhaps we could add a Tip (under Challenge) that says something like - "feel free to explore creative ways of working together to complete this challenge". It's tricky because it seems that it's best when it just happens organically without prompting, so I'm hesitant to add anything at all. What do you think?

Reply:
I think it's good to add a tip. It lets participants know that there is flexibility and they can get creative.

Reply:
+1

Reply:
+1